include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -667,7 +667,7 @@ template <typename _BackendTag, typename _ExecutionPolicy, typename _R, typename
 std::ranges::range_difference_t<_R>
 __pattern_count(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec, _R&& __r, const _T& __value, _Proj __proj)
 {
-    __count_fn_pred<_T> __pred{__value};
+    oneapi::dpl::__internal::__count_fn_pred<_T, _Proj> __pred{__value, __proj};
```

Comment:
__count_fn_pred is defined utils.h, and it is not included. The same applies to another in algorithm_ranges_impl.h.

Reply:
Ok. The structure is pretty rigid, so I will not argue here.